src/Microsoft.ApplicationInsights/TelemetryClientExtensions.cs

```diff
@@ -53,7 +54,33 @@ public static IOperationHolder<T> StartOperation<T>(this TelemetryClient telemet
 
             if (string.IsNullOrEmpty(operationTelemetry.Context.Operation.Id) && !string.IsNullOrEmpty(operationId))
             {
-                operationTelemetry.Context.Operation.Id = operationId;
+                var isActivityAvailable = ActivityExtensions.TryRun(() =>
```

Comment:
we never bothered with making StartOperation work without Activity (it does not work today) - I don't think we should start now. 

The motivation is: from C# noone expects to call AI SDK without installing nugets and depednencies. From script like powershell (which was one scenario that we wanted to address wihtout DiagnosticSource.dll) it's extremely hard to call generic methods, so the expectation is: noone will call startoperation from powershell and no reason to bother to make it resilent for invalid usage,

Reply:
ah, ok, we bothered in some scenarios. I feel quite bad that we overcomplicate code so much just to satisfy some illegal scenario....

Reply:
The standalone Unittest was doing tc.StartOperation...